09_json/alextmz/pkg/puppy/types_test.go

```diff
@@ -0,0 +1,64 @@
+package puppy
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type typetest map[string]struct {
+	obj Puppy
+	jsn string
```

Comment:
sorry, I should have picked up on this earlier.
I'd probably rename this type as:

```go
type testcases map[string]struct {
	puppy Puppy
	puppyJSON string
}
```

Reply:
Done